.github/workflows/pull_requests.yml

```
@@ -0,0 +1,31 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches: [ main ]
+
+## We are cancelling previously triggered workflow runs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
```

Doesn't really matter in this CI, but you can use `${{ github.ref }}` across all kinds of CI files.

[Vapor has been using](https://github.com/vapor/vapor/blob/ebbe71c89aa1b76a0920277760b12be7a2ec7c70/.github/workflows/test.yml#L3) the same thing for a long while and it's been working without fail:

```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```